nbs/methods.ipynb

```diff
@@ -13,7 +13,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Let's also add references for the MinTrace (Wickramasuriya) and ERM (Taieb) methods.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
I think the references should be there already if im not mistaken

Reply:
Ah yes they are indeed missing will add them